pkg/bin/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# TODO:
+# cross-platform dependancy check: clang, perl, ca-certificates (?) others?
+openssl_release="OpenSSL_1_1_0g"
+
+
+wget https://github.com/openssl/openssl/archive/$openssl_release.tar.gz &&
+tar -xf $openssl_release.tar.gz &&
+cd openssl-$openssl_release &&
+CC=clang ./config no-async &&
+make &&
+mkdir lib &&
+cp libcrypto.a libssl.a lib/ &&
+cd ../../.. &&
+OPENSSL_DIR=$PWD/pkg/bin/openssl-$openssl_release/ cargo +nightly build --release
```

Comment:
We need `--target=x86_64-unknown-linux-musl` here, otherwise it won't be a statically linked binary.

Reply:
Also, `OPENSSL_DIR` should be `$PWD/openssl-$openssl_release`.